packages/cannon-worker-api/src/cannon-worker-api.ts

```diff
@@ -75,6 +83,7 @@ export class CannonWorkerAPI extends EventEmitter {
     broadphase = 'Naive',
     defaultContactMaterial = { contactEquationStiffness: 1e6 },
     gravity = [0, -9.81, 0],
+    frictionGravity,
```

Comment:
Please specify a default value and keep the ordering alphabetical.

Reply:
Aye aye on the ordering! On providing a default: the expected default on `cannon-es` for this property is `undefined`. Would you like me to assign `frictionGravity = undefined` here to at least make it clearer to readers? Would a little comment be welcomed?

Reply:
Yes, please assign `undefined` as the default if that's what it is.